requirements.txt

```diff
@@ -1,13 +1,13 @@
 coverage==3.7.1
 coveralls==0.5
-Django==1.7.4
+Django==2.0.0
 docopt==0.6.2
 flake8==3.4.1
 psycopg2==2.7.3
 PyYAML==3.11
 requests==2.7.0
-selenium==2.53.1
-phonenumbers==7.2.6
+selenium==3.8.0
+phonenumbers==8.8.8
 django-cities-light==3.2.0
```

Comment:
You would have to upgrade `django-cities-light` version to 3.3.0. What happens when you try to setup VMS in a new virtual environment?

Reply:
During setup in a virtual environment even with 3.3.0 version basically 2 types of errors are displayed:
In Django 2.0 'on_delete' is a necessary argument while defining fields of a model.
The old django.conf.urls.include() function is now to be imported from django.urls.

I have checked their repository they are already working on it https://github.com/yourlabs/django-cities-light/issues/166
Same is the case with django braces.